src/Library/demos/Level Bars/main.blp

```diff
@@ -0,0 +1,130 @@
+using Gtk 4.0;
+using Adw 1;
+Adw.Clamp{
+  Box {
+    orientation: vertical;
+    margin-top: 24;
+
+    Label {
+      label: "Level Bars";
+      margin-top: 6;
+      margin-bottom: 6;
+
+      styles [
+        "title-1",
+      ]
+    }
+
+    Label {
+      label: _("A bar widget used as a level indicator");
+      margin-top: 6;
+      margin-bottom: 24;
+    }
+
+    Box {
+      orientation: vertical;
+      vexpand: true;
+      spacing: 18;
+      margin-start: 18;
+      margin-end: 18;
+
+      Label {
+        label: _("Continuous Level Bar");
+      }
+
+      Box{
+        spacing:18;
+        halign: start;
+
+        Label {
+          label: _("Current Battery Level");
+        }
+
+        SpinButton spin_button{
+          digits:1;
+          adjustment: Gtk.Adjustment {
+            lower: 0;
+            upper: 100;
+            value:100;
+            step-increment: 5;
+          };
+        }
+      }
+
+      Box {
+        orientation: vertical;
+        spacing:12;
+
+        LevelBar bar_continuous{
+            margin-end:12;
+            margin-start:12;
+            margin-top:24;
+            min-value:0;
+            max-value:100;
+            mode: continuous;
+            value: bind spin_button.value;
```

Comment:
yes this makes it difficult to grasp for first-timer

and the LevelBar demo should work when copy-pasted somewhere else
use a fixed value like 75%

Reply:
The only problem is I won't be able to synchronize the ``percentage_label`` and level bar value when the UI is changed. They'll have to run the demo again if that's fine.

Reply:
Do we need the % label?

Reply:
Well I wanted it to make it look like a battery widget similar to GNOME's to show how it's actually being used. I can remove it if you want.

Reply:
It's not about what I want - it's about pros and cons

Given the goal of Workbench/the project it's more important for the user to understand and be able to copy paste the Blueprint quickly rather than for the UI to show a spin button and label.

It still looks clear/good IMHO:

![image](https://user-images.githubusercontent.com/19673/226482762-a1022565-cc6f-43c8-b8e1-89a81ef2ed27.png)

Reply:
Hmm I'll keep that in mind 👍🏾 

I guess it's trying to find the balance between trying to make the demo easily copy-pastable and free of bloat while also making sure it looks good.